### PR TITLE
Disable failing external executor test on SNP

### PR DESCRIPTION
**Pull request by DomAyre**

### CODE REVIEW

File: tests/external_executor/external_executor.py [hunk 1]
```diff
@@ -405,10 +406,10 @@ def run(args):
...
 
         network = test_executor_registration(network, args)
-        network = test_put_get(network, args)
-        network = test_simple_executor(network, args)
-        network = test_parallel_executors(network, args)
-        network = test_streaming(network, args)
+        # network = test_put_get(network, args)
+        # network = test_simple_executor(network, args)
+        # network = test_parallel_executors(network, args)
+        # network = test_streaming(network, args)
```
Comment (eddyashton) [inline tests/external_executor/external_executor.py:412]: These should be re-enabled

File: src/apps/external_executor/executor_code_id.h [hunk 2]
```diff
@@ -50,6 +50,12 @@ inline std::pair<grpc_status, std::string> verification_error(
       return std::make_pair(
         GRPC_STATUS_UNAUTHENTICATED,
         "Quote report data does not contain node's public key hash");
+    case ccf::QuoteVerificationResult::FailedHostDataDigestNotFound:
+      return std::make_pair(
+        GRPC_STATUS_UNAUTHENTICATED, "Quote does not contain host data");
+    case ccf::QuoteVerificationResult::FailedInvalidHostData:
+      return std::make_pair(
+        GRPC_STATUS_UNAUTHENTICATED, "Quote host data is not authorised");
```
Comment (eddyashton) [inline src/apps/external_executor/executor_code_id.h:58]: These changes look unrelated?

File: tests/external_executor/external_executor.py (hunk 1 above)
Comment (DomAyre) [reply tests/external_executor/external_executor.py:412]: Oops

File: src/apps/external_executor/executor_code_id.h (hunk 2 above)
Comment (DomAyre) [reply src/apps/external_executor/executor_code_id.h:58]: Yeah they are, but I figured they were worth getting in anyway

File: tests/external_executor/external_executor.py (hunk 1 above)
Comment (DomAyre) [reply tests/external_executor/external_executor.py]: Oops

**Review by eddyashton — APPROVED** [review]

File: src/apps/external_executor/executor_code_id.h (hunk 2 above)
Comment (MahatiC) [reply src/apps/external_executor/executor_code_id.h:58]: I think the way this is implemented will change in the near future, but for now this is ok.

**Review by MahatiC — APPROVED** [review]

File: tests/external_executor/external_executor.py [hunk 3]
```diff
@@ -101,7 +101,10 @@ def register_new_executor(node, network, message=None):
     return executor_credentials
 
 
-@reqs.description("Register an external executor")
+@reqs.description(
+    "Register an external executor (Disabled on SNP due to UNKNOWN RPC failures)"
```
Comment (jumaffre) [inline tests/external_executor/external_executor.py:105]: Could you create a Github issue for this and link it here? It becomes hard to keep track of special cases like this one otherwise.

Comment (DomAyre) [reply tests/external_executor/external_executor.py:105]: Done #4505

**Review by jumaffre — APPROVED** [review]

**Review by achamayou — APPROVED** [review]
